.codesandbox/ci.json

```diff
@@ -1,4 +1,9 @@
 {
-    "sandboxes": ["new", "vanilla"],
+    "buildCommand": "build-ci",
+    "sandboxes": ["@eightfold.ai/octuple", "/storybook-static"],
```

Comment:
Looks like this config might need to be updated.

Reply:
Seems to be expecting a package.json in storybook-static. Unfortunately, documentation is on this config is rather light
https://codesandbox.io/docs/ci#configuration